cluster/gce/gci/configure-helper.sh

```diff
@@ -3376,6 +3396,9 @@ function main() {
     log-wrap 'StartKubeScheduler' start-kube-scheduler
     log-wrap 'WaitTillApiserverReady' wait-till-apiserver-ready
     log-wrap 'StartKubeAddons' start-kube-addons
+    if [[ "${INSTALL_PDCSI:-}" == "true" ]]; then
```

Comment:
`${INSTALL_PDCSI}` should have a value at this point, probably `:-` is not needed

Reply:
I think it doesn't hurt to have it just in case somehow the var is not set. also my guess is that this somehow became convention because most of the time people use `:-` to provide some default